cli/internal/context/context.go

```diff
@@ -143,6 +144,7 @@ func WithGraph(repoRoot fs.AbsolutePath, rootPackageJSON *fs.PackageJSON, cacheD
 		}
 
 		// Get the workspaces from the package manager.
+		// workspaces are absolute paths
```

Comment:
That moment when you finally find where you're going to stop pulling the thread.